wordle-web/src/assets/main.css

```diff
@@ -6,6 +6,20 @@ a,
   color: hsla(160, 100%, 37%, 1);
   transition: 0.4s;
 }
+.titleClass {
+  color: black;
+  font-family: 'Helvetica Neue';
+  font-weight: bolder;
+  font-size: 30px;
+}
+.marvel-theme {
+  background-color: #ef9a9a;
+  color: #8c9eff;
+}
+.dc-theme {
+  background-color: #ffd600;
+  color: #000000;
+}
```

Comment:
```suggestion
```
I don't think these are being used anywhere.

Reply:
They are being used in our themes but thanks for the heads up cause I think it's written in css